class.jetpack-jitm.php

```diff
@@ -321,7 +321,7 @@ function backups_updates_msg() {
 			<a href="#" data-module="vaultpress" class="dismiss"><span class="genericon genericon-close"></span></a>
 
 			<div class="jp-emblem">
-				<?php echo self::get_jp_emblem(); ?>
+				<?php echo Jetpack::get_jp_emblem(); ?>
```

Comment:
Agreed, however, at this point we're just getting things turned back on as they were, there's going to be a much deeper rewrite coming soon